<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductDialog</class>
 <widget class="QDialog" name="AddProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(102, 16, 53);
}
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: '#ebebeb';
 background: '#75103c';
 color: '#e8e8e8';
 border: 4px solid '#1a1818';
 border-radius: 15px;
}

QLabel::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prodNameBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>158</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 16pt &quot;HP Simpified Jpan&quot;;
	border: 4px solid rgb(26, 24, 24);
	border-radius: 10px;
	padding-left: 10px;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Expire</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>243</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
	border: 4px solid rgb(26, 24, 24);   
	border-radius: 18px;
	padding-left: 10px;
	padding-top: 4px;
	padding-bottom: 4px;
	font: 700 16pt &quot;Roboto&quot;;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>337</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 16pt &quot;HP Simpified Jpan&quot;;
	border: 4px solid rgb(26, 24, 24);
	border-radius: 10px;
	padding-left: 10px;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>428</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 16pt &quot;HP Simpified Jpan&quot;;
	border: 4px solid rgb(26, 24, 24);
	border-radius: 10px;
	padding-left: 25px;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(235, 235, 235)
}</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="qntyBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	border: 4px solid rgb(26, 24, 24);   
	border-radius: 18px;
	padding-left: 25px;
	padding-top: 4px;
	padding-bottom: 4px;
	font: 700 16pt &quot;Roboto&quot;;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
}

QPushButton::hover {
border: 5px solid '#1a1818';
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Save your informations</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
